contracts/tokenbridge/arbitrum/L2AtomicTokenBridgeFactory.sol

```diff
@@ -189,8 +197,12 @@ contract L2AtomicTokenBridgeFactory {
             ITransparentUpgradeableProxy(canonicalCustomGateway), customGatewayLogicAddress
         );
 
+        // init logic contract with dummy values
+        address dead = address(0x000000000000000000000000000000000000dEaD);
```

Comment:
We could make this a const var?

Reply:
Right, having it as a constant increases contract size a bit more compared to doing it this way. But actually both approaches fit the size limits, so I've made it a constant as it's nicer - https://github.com/OffchainLabs/token-bridge-contracts/pull/54/commits/4d0f93bb60a1791bce6341fbeaf66f25bdd4472f